src/test/java/com/wikia/webdriver/common/core/imageutilities/ImageComparison.java

```diff
@@ -74,6 +73,17 @@ public boolean isColorImage(BufferedImage image, Color color, int accuracy) {
     return true;
   }
 
+  public boolean isMonocolorImage(BufferedImage image) {
```

Comment:
Does this method return true, if given image has only one colour in its entire area ?
Please add a javadoc that clarifies that